packages/utils/src/types/actor-type.utils.ts

```diff
@@ -0,0 +1,15 @@
+import type { ActorMethod } from "@dfinity/agent";
+
+export type ServiceParam<S, K extends keyof S> = S[K] extends ActorMethod<
```

Comment:
super nice!

not for this PR but, I ask my self if that handy types should not be part of agent-js in the future

Reply:
Yeah I agree it shouldn't belong here in the long run. I think it could be good in the candid JS library. That library is currently something that I have 0 knowledge of right now, but it's definitely on my to-do list to get my hands dirty with it at some point.